options.c

```diff
@@ -259,7 +260,7 @@ static optionDescription LongOptionDescription [] = {
  {1,"  --help"},
  {1,"       Print this option summary."},
  {1,"  --if0=[yes|no]"},
- {1,"       Should C code within #if 0 conditional branches be parsed [no]?"},
+ {1,"       Should C/C++ code within #if 0 conditional branches be parsed [no]?"},
```

Comment:
I'd rather remove the language detail like you did in `--undef` description than refer to "C/C++" (which is probably the greatest lie about C and C++).  Moreover, I think there are other languages that are affected (D? Verilog?), too :)